app/lib/theme/components/FileComponent.tsx

```diff
@@ -209,23 +238,79 @@ const FileComponent: React.FC<FileComponentProps> = ({
         {error && <ErrorMessage error={error} fileTypes={fileTypes} />}
       </StyledDetails>
       <StyledDetails>{statusLabel}</StyledDetails>
-      <div>
-        <StyledButton
-          addBottomMargin={wrap}
-          id={`${id}-btn`}
-          onClick={(e: React.MouseEvent<HTMLInputElement>) => {
-            e.preventDefault();
-            handleClick();
-          }}
-          variant={buttonVariant}
-          disabled={loading || disabled}
-        >
-          {loading ? (
-            <LoadingSpinner color={isSecondary ? '#000000' : '#fff'} />
-          ) : (
-            buttonLabel()
-          )}
-        </StyledButton>
+      <div
+        style={
+          useFileDate
+            ? { display: 'flex', flexDirection: 'row', alignItems: 'center' }
+            : {}
+        }
+      >
+        {useFileDate && (
+          <div style={{ height: '100%' }}>
+            <h4 style={{ marginBottom: '8px' }}>{`${fileDateTitle}`}</h4>
+            <LocalizationProvider
+              localeText={
+                enUS.components.MuiLocalizationProvider.defaultProps.localeText
+              }
+              dateAdapter={AdapterDayjs}
+            >
+              <StyledDatePicker
+                id={id}
+                sx={getStyles(false)}
+                disabled={false}
+                readOnly={false}
+                onChange={(d: Date | null) => {
```

Comment:
When I choose a date on the date picker, I get a an error of `TypeError: value.isBefore is not a function`. I remember this is from not using dayjs when setting the date I think was the problem.

Reply:
This should be good, give it another try when you get a chance :)